target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -1005,7 +1099,9 @@ TARGET_DEVICES += AD1018-SPI_flash
 ### SFR ###
 define Device/NEUFBOX4-SER
   $(Device/bcm63xx)
-  DEVICE_TITLE := SFR Neufbox4 (Sercomm)
+  DEVICE_VENDOR := SFR
+  DEVICE_MODEL := Neufbox4
+  DEVICE_VARIANT := Sercomm
```

Comment:
Is this a branding or why is the vendor used as variant?

Reply:
I think they switched the manufacturer which resulted in different hardware.

Reply:
Hmm, if nobody has a substantially better idea, I would just keep your current version.

Reply:
ok